manifests/00-custom-resource-definition-internal.yaml

```diff
@@ -7,6 +7,7 @@ metadata:
     include.release.openshift.io/ibm-cloud-managed: "true"
     include.release.openshift.io/self-managed-high-availability: "true"
     include.release.openshift.io/single-node-developer: "true"
+    capability.openshift.io/name: Ingress
```

Comment:
This is going away, hopefully in the next api module update.  https://redhat-internal.slack.com/archives/CBWMXQJKD/p1690996863907609

Reply:
yes I added the latest merge commit of openshift/api